spec/support/capybara_helpers.rb

```diff
@@ -158,5 +158,25 @@ def draft_age_passported_application(case_type:) # rubocop:disable Metrics/Metho
     fill_in('Telephone number', with: '07715339488')
 
     stub_request(:post, 'http://datastore-webmock/api/v1/applications')
+      .with { |req|
+      @submitted_body = req.body
+      true
+    }
+  end
```

Comment:
Here  we capture the submitted json, so we can use it to create the returned application. 
(And in doing so, we also test that re-hydration works based on the submitted application 👏 )